hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAsyncClusterAdminApi.java

```diff
@@ -139,14 +173,14 @@ public void testAllClusterOnlineConfigChange() throws IOException {
 
   private void replaceHBaseSiteXML() throws IOException {
     // make a backup of hbase-site.xml
-    Files.copy(cnfPath, cnf3Path, StandardCopyOption.REPLACE_EXISTING);
+    Files.copy(newCnfPathes.get(0), newCnfPathes.get(2), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
nit for other readers: since you're here cleaning up, can you use descriptive names for these variables? Instead of `cnfPath{1,2,3}` or this array of paths, how about
* `rsConfigFilePath` instead of `cnfPath`
* `overwriteFilePath` instead of `cnf1Path`
* `rsConfigFileBackup` instead of `cnf3Path`

IMHO, descriptive variable names are better than comments. Just a suggestion :)

Reply:
Sure, will do, :)